driver/task.go

```diff
@@ -91,7 +91,11 @@ func NewTask(conf TaskConfig) (*Task, error) {
 	// Load all variables from passed in variable files
 	loadedVars := make(hcltmpl.Variables)
 	for _, vf := range conf.VarFiles {
-		tfvars, err := tftmpl.LoadModuleVariables(vf)
```

Comment:
This is in the wrong commit, should be part of the refactor. I'll fix this before mering